indico/modules/categories/blueprint.py

```diff
@@ -26,6 +27,10 @@
 
 # Management
 _bp.add_url_rule('/manage/', 'manage_content', RHManageCategoryContent)
+_bp.add_url_rule('/manage/settings/', 'manage_settings', RHManageCategorySettings, methods=('POST', 'GET'))
+_bp.add_url_rule('/manage/settings/upload_icon', 'upload_icon', RHSettingsIconUpload, methods=('POST',))
+_bp.add_url_rule('/manage/settings/delete_icon', 'delete_icon', RHSettingsIconDelete, methods=('DELETE',))
```

Comment:
yeah, `DELETE .../delete-icon` is a actually weird, it sounds like you delete a 'delete-icon' thing ;)